app/index.html

```diff
@@ -72,11 +72,11 @@
                     <li><a target="_blank" ng-href="{{FACEBOOK_URL}}"><span class="icon-facebook"></span></a></li>
                     <li><a target="_blank" ng-href="{{TUMBLR_URL}}"><span class="icon-tumblr"></span></a></li>
                     <li><a target="_blank" ng-href="{{GITHUB_URL}}"><span class="icon-github"></span></a></li>
-                    <li ng-click="trackEvent('donate')" class="bc"><div class="coinbase-button"
+                    <!--<li ng-click="trackEvent('donate')" class="bc"><div class="coinbase-button"
                         data-code="deeffa56195e84b5b9991ce7da426c9f"
                         data-button-style="none"></div><div
                         class="bc-link"><img
-                        src="/images/bitcoin-logo-cyan.png"></div></li>
+                        src="/images/bitcoin-logo-cyan.png"></div></li>-->
```

Comment:
I actually figured out why this is broken, but I'll merge your changes and just uncomment the section before I deploy to production. Thanks!